api/server/server.go

```diff
@@ -5,11 +5,33 @@ import (
 	"log"
 	"net"
 
+	"fmt"
```

Comment:
**Totally** nitpicking here, but but "fmt" and "os" should get moved up to be by "log" and "net".